tests/src/test_class.cpp

```diff
@@ -0,0 +1,28 @@
+#include <dbg_macro/dbg.h>
+
+template <typename L, typename R>
+void assert_eq(const L& lhs, const R& rhs) {
```

Comment:
I'm fine with splitting the tests, but we definitely shouldn't repeat the implementation of `assert_eq` in each of the files. Can we please move it to a `test.h` or `assert_eq.h` header?

Reply:
Yeah, I splitted the tests for convenience. I had the same idea about creating `test.h`! I just wanted to hear it from you :+1:. I will move those functions to `test.h`.

Reply:
Done!